### PR TITLE
Removes Hygiene

### PR DESCRIPTION
**Pull request by Wilchenx**
## About The Pull Request

Removes hygiene
image

NEET quirk has not been removed
Miasma has not been removed

## Why It's Good For The Game

The game no longer smells bad.

## Changelog
:cl:
del: hygiene is gone 🦀 
/:cl:

### CODE REVIEW

**Comment by Kapu1178** [comment]
mechanic nobody likes and bogs down Life()

**Comment by Wilchenx** [comment]
> mechanic nobody likes

I don't know about downstream erp servers but atleast that's true for beestation

**Comment by ike709** [comment]
Good PR

**Comment by AnCopper** [comment]
image

**Comment by Archanial** [comment]
how will gamers recover?

**Comment by R3pID** [comment]
istink

**Comment by PerishedFraud** [comment]
What's this? Actually removing a _bad_ mechanic? Well I'll be.

**Comment by williamcll** [comment]
uh oh, stinky!

**Comment by r1ks-iwnl** [comment]
#4068 should also be removed

**Comment by Wilchenx** [comment]
> #4068 should also be removed

https://github.com/BeeStation/BeeStation-Hornet/pull/4528

**Comment by Wilchenx** [comment]
btw i didn't test this 😈

**Comment by utilbird** [comment]
bless

File: code/controllers/subsystem/processing/quirks.dm [hunk 1]
```diff
@@ -17,7 +17,7 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 	if(!quirks.len)
 		SetupQuirks()
 
-	quirk_blacklist = list(list("Blind","Nearsighted"),list("Jolly","Depression","Apathetic","Hypersensitive"),list("Ageusia","Vegetarian","Deviant Tastes"),list("Ananas Affinity","Ananas Aversion"),list("Alcohol Tolerance","Light Drinker"),list(list("Neat","NEET")))
+	quirk_blacklist = list(list("Blind","Nearsighted"),list("Jolly","Depression","Apathetic","Hypersensitive"),list("Ageusia","Vegetarian","Deviant Tastes"),list("Ananas Affinity","Ananas Aversion"),list("Alcohol Tolerance","Light Drinker"),list(list("NEET")))
```
Comment (KubeRoot) [inline code/controllers/subsystem/processing/quirks.dm]:
```suggestion
	quirk_blacklist = list(list("Blind","Nearsighted"),list("Jolly","Depression","Apathetic","Hypersensitive"),list("Ageusia","Vegetarian","Deviant Tastes"),list("Ananas Affinity","Ananas Aversion"),list("Alcohol Tolerance","Light Drinker"))
```

No point having just "NEET" there.

Comment (Wilchenx) [reply code/controllers/subsystem/processing/quirks.dm]: thanks for the catch

**Comment by ike709** [comment]
Failing the linter.

**Comment by Wilchenx** [comment]
Should be good for test merge if it wasn't done yet.

**Review by Archanial — APPROVED** [review]

**Pull request merged**